Doc/library/bdb.rst

```diff
@@ -311,16 +361,18 @@ The :mod:`bdb` module also defines two classes:
 
    .. method:: get_stack(f, t)
 
-      Get a list of records for a frame and all higher (calling) and lower
-      frames, and the size of the higher part.
+      Return a list of (frame, lineno) tuples in a stack trace and a size.
```

Comment:
The output is ((stack trace), size).  So, it's a tuple of the form (list of tuples, int).   I've added the comma before "and a size".